src/test/java/uk/gov/pay/api/exception/mapper/CreateChargeExceptionMapperTest.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.pay.api.exception.mapper;
+
+import org.junit.Before;
+import org.junit.Test;
+import uk.gov.pay.api.exception.ConnectorResponseErrorException.ConnectorErrorResponse;
+import uk.gov.pay.api.exception.CreateChargeException;
+import uk.gov.pay.api.model.PaymentError;
+import uk.gov.pay.commons.model.ErrorIdentifier;
+
+import javax.ws.rs.core.Response;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static uk.gov.pay.api.model.PaymentError.Code.CREATE_PAYMENT_MANDATE_STATE_INVALID;
+import static uk.gov.pay.api.model.PaymentError.aPaymentError;
+
+public class CreateChargeExceptionMapperTest {
+
+    private CreateChargeExceptionMapper mapper;
+    
+    @Before
+    public void setUp() {
+        mapper = new CreateChargeExceptionMapper();
+    }
+    
+    @Test
+    public void shouldThrow409_whenMandateStateInvalid(){
+        Response response = mock(Response.class);
```

Comment:
Usually we’d put a mock like this in a field with an annotation:

```java
@Mock private Response mockResponse;
```

You’ll need to add a class-level `@RunWith(MockitoJUnitRunner.class)` annotation for this to work.